CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
 - Just run CI on the oldest + newest supported Python versions ([#2074](https://github.com/ewels/MultiQC/pull/2074))
 - Strict mode: rename `config.lint` to `config.strict`, crash early on module or template error. Add `MULTIQC_STRICT=1` ([#2101](https://github.com/ewels/MultiQC/pull/2101))
 - Trigger changelog entry addition on PR creation, in addition to an explicit comment to multiqc-bot ([#2102](https://github.com/ewels/MultiQC/pull/2102))
+- Added support for fature barcode data i.e Antibody (CITE-seq) in cellranger module ([#1909](https://github.com/ewels/MultiQC/issues/1909))
+- Cellranger count module updated to parse Gene Expression + Antibody Capture summary files ([#2118](https://github.com/ewels/MultiQC/pull/2118))
```

Comment:
The bot added a separate changelog entry, so it's duplicated now. Note also that for module updates, it's nice to start the PR title with the module name followed by colon, so the bot puts the entry under the "Module updates" section:

### Module updates
- **Cellranger**: Count module updated to parse Gene Expression + Antibody Capture summary files ([#2118](https://github.com/ewels/MultiQC/pull/2118))

If you could remove the entries above and add one under "Module updates" as above, would be awesome!

Reply:
I just pushed that into the branch :)

Reply:
Thanks! I'll keep this in mind for future PRs.